<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgResoluciones</class>
 <widget class="QDialog" name="DlgResoluciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="twResoluciones">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtResolucion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>681</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="twTemas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btQuitarTema">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitar tema</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAnadirTema">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>AÃ±a&amp;dir tema</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCerrar">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cerrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
